Formula/tunnel.rb

```diff
@@ -19,8 +19,6 @@ def install
   end
 
   test do
-    system bin/"tunnel", "start", "8080"
-    system bin/"tunnel", "kill"
-    assert_predicate testpath/".tunnel/daemon.log", :exist?
+    system bin/"tunnel", "ping"
```

Comment:
Why modify the test? What does the new test do?

Reply:
In v0.4.0 we can use `ping` to test the local program and remote server's health instead making a persistent connection and killing the process.